packs/qualys/actions/add_host.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: add_host
+runner_type: python-script
+description: Add an IP of a host to the Qualys registry
+enabled: true
+entry_point: add_host.py
+parameters:  
+  host:
+    type: string
+    description: The host IP, or a list seperated with commas
```

Comment:
If this also supports multiple hosts I would prefer this argument to be of a type `array` and join the array into a string inside the action, if required.